packages/clients/dish/src/plugins/DishExportMap/DishExportMap.vue

```diff
@@ -0,0 +1,235 @@
+      this.exportMapAsPdfUrl ||= `${this.configuration.dishExportMap.internalHost}/Content/Objekt/Kartenausgabe.aspx`
+    },
+    showRectangleAndDialog() {
+      if (
+        !this.transformedCoordinate ||
+        this.transformedCoordinate.length === 0
+      ) {
+        return
+      }
+      if (this.overlay) {
+        this.showOverlay = true
+        this.dialog = true
+      }
+    },
+    setTitle() {
+      if (this.overlay && this.hasObjectProperties) {
+        const objectId = this.currentProperties.objektid
+        const address = [
+          this.currentProperties.strasse,
+          this.currentProperties.hausnummer,
+          this.currentProperties.hausnrzusatz,
+        ]
+          .join(' ')
+          .trim()
+        this.title = `${this.currentProperties.gemeinde}, ${address}, ${this.currentProperties.objektansprache}, ONR ${objectId}`
+        this.overlay.setPosition(this.transformedCoordinate)
+      }
+    },
+    getRectangleCoordinates() {
+      if (this.transformedCoordinate) {
+        const centerPixel = this.map.getPixelFromCoordinate(
+          this.transformedCoordinate
+        )
+        const topLeftPixel = [
+          centerPixel[0] - this.rectangleWidth / 2,
+          centerPixel[1] - this.rectangleHeight / 2,
+        ]
+        const bottomRightPixel = [
+          centerPixel[0] + this.rectangleWidth / 2,
+          centerPixel[1] + this.rectangleHeight / 2,
+        ]
+        const topLeft = this.map.getCoordinateFromPixel(topLeftPixel)
+        const bottomRight = this.map.getCoordinateFromPixel(bottomRightPixel)
+
+        return {
+          xMin: topLeft[0],
+          xMax: bottomRight[0],
+          yMin: topLeft[1],
+          yMax: bottomRight[1],
+        }
+      }
+      console.error('Center coordinates are undefined.')
+    },
+    getPrintParams() {
+      const backgroundLayer = this.getBackgroundLayer()
+      const bbox = this.getRectangleCoordinates()
+      return {
+        NewTab: true,
+        objektueberschrift: this.title,
+        masssstab: this.scaleValue,
+        printApproach: 'scale',
+        printRequester: 'client',
+        id: this.currentProperties.objektid,
+        xPrint: 18,
+        yPrint: 20,
+        scale: this.scaleValue,
+        xMin: bbox?.xMin,
+        yMin: bbox?.yMin,
+        xMax: bbox?.xMax,
+        yMax: bbox?.yMax,
+        xCenter: this.transformedCoordinate[0],
+        yCenter: this.transformedCoordinate[1],
+        mapSRS: this.configuration.epsg,
+        urlHintergrund: `${backgroundLayer.url}?`,
+        LayerNameHintergrund: backgroundLayer.layers,
+        VersionHintergrund: '1.1.1',
+        ProxyHintergrund: 'y',
+        urlWMS: `${this.wmsLayerUrl}?`,
+        VersionWMS: '1.1.1',
+        LayerNameWMS:
+          '0,9,1,10,2,11,3,12,4,13,25,27,24,26,6,15,19,30,20,31,21,32,22,33,23,34,29,36,28,35',
+        urlWFS: `${this.wfsLayerUrl}?`,
+        VersionWFS: '1.1.0',
```

Comment:
As before, this should not be static as the configuration for the layer may differ in the future.